specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2020-11-01-preview/containerregistry.json

```diff
@@ -5579,13 +5579,66 @@
           }
         },
         "systemData": {
-          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/systemData",
+          "$ref": "#/definitions/SystemData",
           "description": "Metadata pertaining to creation and last modification of the resource.",
           "type": "object",
           "readOnly": true
         }
       },
       "x-ms-azure-resource": true
+    },
+    "SystemData": {
```

Comment:
This mirrors the SystemData definition in common-types.
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/common-types/resource-management/v2/types.json#L445..L490